Cannot load implementation hint 'com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule

Possible user or docs error

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (enforce) on project enforcertest: Unable to parse configuration of mojo org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce for parameter rules: Cannot load implementation hint 'com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule' -> [Help 1]

--- a/enforcer-rules/README.md
+++ b/enforcer-rules/README.md
@@ -35,6 +35,8 @@ Add the following plugin configuration to your `pom.xml`:
         <executions>
           <execution>
             <id>enforce</id>
+            <!-- Important! Should run after compile -->
+            <phase>process-classes</phase>
             <goals>
               <goal>enforce</goal>
             </goals>

--- a/enforcer-rules/README.md
+++ b/enforcer-rules/README.md
@@ -42,7 +42,7 @@ Add the following plugin configuration to your `pom.xml`:
             </goals>
             <configuration>
               <rules>
-                <banLinkageErrors
+                <LinkageCheckerRule
                     implementation="com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule"/>
               </rules>
             </configuration>
@@ -55,10 +55,10 @@ Add the following plugin configuration to your `pom.xml`:
 For a BOM project, set `dependencySection` element to `DEPENDENCY_MANAGEMENT`.
 
 ```xml
-  <banLinkageErrors
+  <LinkageCheckerRule
       implementation="com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule">
       <dependencySection>DEPENDENCY_MANAGEMENT</dependencySection>
-  </banLinkageErrors>
+  </LinkageCheckerRule>
 ```
 
 To suppress linkage errors that are not [_reachable in the class reference graph_](
@@ -66,27 +66,19 @@ To suppress linkage errors that are not [_reachable in the class reference graph
 dependencies of the project, set `reportOnlyReachable` element to `true`. (default: `false`).
 
 ```xml
-  <banLinkageErrors
+  <LinkageCheckerRule
       implementation="com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule">
       <reportOnlyReachable>true</reportOnlyReachable>
-  </banLinkageErrors>
+  </LinkageCheckerRule>
 ```
 
 If a violation should not fail the build, set `level` element to `WARN`:
 
 ```xml
-  <banLinkageErrors
+  <LinkageCheckerRule
       implementation="com.google.cloud.tools.dependencies.enforcer.LinkageCheckerRule">
       <level>WARN</level>
-  </banLinkageErrors>
-```
-
-# Run
-
-Enforcer rules are part of the `validate` lifecycle in Maven.
-
-```
-$ mvn validate
+  </LinkageCheckerRule>
 ```
 
 ## Debug

--- a/enforcer-rules/README.md
+++ b/enforcer-rules/README.md
@@ -29,7 +29,7 @@ Add the following plugin configuration to your `pom.xml`:
           <dependency>
             <groupId>com.google.cloud.tools</groupId>
             <artifactId>linkage-checker-enforcer-rules</artifactId>
-            <version>0.1</version>
+            <version>0.1.0</version>
           </dependency>
         </dependencies>
         <executions>

